<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FitGeneratorDemoViewControls</class>
 <widget class="QWidget" name="FitGeneratorDemoViewControls">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>349</width>
    <height>671</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>0</width>
    <height>0</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>QmitkTemplate</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QLabel" name="label">
     <property name="text">
      <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-weight:600;&quot;&gt;Simple demo for a parameter map generator&lt;/span&gt;&lt;br/&gt;Computes the fit of a linear model (model parameters are slope and offset).&lt;/p&gt;&lt;p&gt;Demo steps:&lt;/p&gt;&lt;p&gt;[0. Press &amp;quot;Generate test data&amp;quot;, if you dont want to try your own]&lt;br/&gt;1. Choose the data node that should be used as input. (If you want to select a mask, press &amp;quot;CTRL&amp;quot; and select the mask node.)&lt;br/&gt;2. Press &amp;quot;Start Modelling&amp;quot;&lt;br/&gt;3. Wait and see...&lt;br/&gt;4. The output nodes &amp;quot;slope&amp;quot; and &amp;quot;offset&amp;quot; can be found as child nodes for the chosen input.&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
     <property name="wordWrap">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QGridLayout" name="gridLayout">
     <item row="0" column="0">
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>Selected Time Series:</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QLabel" name="lTimeseries">
       <property name="text">
        <string>None</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="label_4">
       <property name="text">
        <string>Mask (optional):</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QLabel" name="lMask">
       <property name="text">
        <string>None</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QPushButton" name="btnGenerateTestData">
     <property name="text">
      <string>Generate test data</string>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLabel" name="label_3">
       <property name="text">
        <string>Fit name:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="leFitName"/>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QPushButton" name="btnModelling">
     <property name="text">
      <string>Start Modelling</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QTextEdit" name="textEdit"/>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
